mdx_math.py

```diff
@@ -15,14 +15,23 @@ class MathExtension(markdown.extensions.Extension):
     def __init__(self, *args, **kwargs):
         self.config = {
             'enable_dollar_delimiter': [False, 'Enable single-dollar delimiter'],
+            'render_to_span': [False,
+                'Render to span elements rather than script for fallback'],
         }
         super(MathExtension, self).__init__(*args, **kwargs)
 
     def extendMarkdown(self, md, md_globals):
         def handle_match_inline(m):
-            node = markdown.util.etree.Element('script')
-            node.set('type', 'math/tex')
-            node.text = markdown.util.AtomicString(m.group(3))
+            if not self.getConfig('render_to_span'):
+                node = markdown.util.etree.Element('script')
+                node.set('type', 'math/tex')
+                node.text = markdown.util.AtomicString(m.group(3))
+            else:
+                node = markdown.util.etree.Element('span')
```

Comment:
I only use inline math for my purposes. I'm not sure of how standalone math would work, at any rate you couldn't just create a standalone span as it's not a block element. README.md points out that the setting only applies to inline math. Hopefully that's OK?